src/main/java/com/bobocode/bibernate/configuration/DialectResolver.java

```diff
@@ -0,0 +1,27 @@
+package com.bobocode.bibernate.configuration;
+
+import com.bobocode.bibernate.exception.BibernateException;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DialectResolver {
+
+
+    private DialectResolver() {
+    }
+
+    /**
+     * Returns item of {@link SqlDialect} by provided dialect name ignoring case
+     *
+     * @return value of {@link SqlDialect}
+     * @throws BibernateException in case of providing unsupported driver name
+     */
+    public static Dialect resolveDialect(String dialectName) {
```

Comment:
`SqlDialect` should be returned here. If we want to get `Dialect` instance we can call method `getInstnce()` on this `SqlDialect` enum